docs/api/item.md

```diff
@@ -28,6 +32,125 @@ import Basic from '@site/static/usage/v7/item/basic/index.md';
+  dontImage={<img alt="A list that contains several items, each representing a different to-do list. Two counts are placed at the end of each item: One count that states the total number of tasks, and another count that states the total number of tasks that are due today." src={useBaseUrl('img/item/metadata-relevant-dont.jpg')} />}
+/>
+
+<BestPracticeFigure 
+  text="Developers should also consider how important the metadata is. Drawing attention to the metadata may be helpful for the user or it may distract them from the more important information depending on the use case."
+  doText="Prioritize the most important content."
+  cautionText="Prioritized metadata may distract from other important content."
+  doImage={<img alt="A list that contains several items, each representing a different to-do list. A count of how many tasks in each to-do list is placed at the end of each item." src={useBaseUrl('img/item/metadata-relevant-do.jpg')} />}
+  cautionImage={<img alt="A list that contains several items, each representing a different to-do list. A count of how many tasks in each to-do list is placed at the end of each item. However, the count is highlighted in blue which draws the user's attention away from the name of the to-do list." src={useBaseUrl('img/item/metadata-important-caution.jpg')} />}
+/>
+
+In the example below, we are creating two lists with different kinds of metadata. The first list uses [Note](./note) to show how many tasks are in each to-do list.
+
+The second list mimics the iOS Mail app to show an inbox. This list makes use of custom metadata including an "unread message" indicator in the "start" slot as well as a timestamp and custom detail icon in the "end" slot. The "unread message" indicator is highlighted in blue to draw the user's attention to the unread messages, while the timestamp is more subtle.
+
+import Metadata from '@site/static/usage/v7/item/content-types/metadata/index.md';
+
+<Metadata />
+
+### Actions
+
+Actions are interactive elements that do something when you activate them. An item can have multiple actions displayed on a line. However, developers should ensure that each action's tap target is large enough to be usable.
+
+<BestPracticeFigure 
+  text={<>Developers should avoid creating <a href="https://dequeuniversity.com/rules/axe/4.4/nested-interactive">nested interactives</a> which can break the user experience with screen readers. For example, developers should avoid adding a button inside the main content of the Item if the <code>button</code> property is set to <code>true</code>. Additionally, developers should use the <a href={useBaseUrl('api/item-sliding')}>Item Sliding</a> component if they need to show more than two actions.</>}
+  doText={<>Avoid nested interactives and put multiple items in <a href={useBaseUrl('api/item-sliding')}>Item Sliding</a> that can be revealed by swiping on the Item.</>}
+  dontText="Don't put too many actions on a single item."
```

Comment:
I'm not sure we should make this change. You can place too many actions on the same item even without creating nested interactives. This line implies that it's ok to put one action in the form of a nested interactive

Reply:
Maybe there is another way we could say it then. To me it read funny because we are saying that you _can_ put actions just do it in the form of a sliding item instead.

Reply:
Ok cleaned this up in https://github.com/ionic-team/ionic-docs/pull/3123/commits/596ba7c9775fdfe26def5415e1ba596c66a5822d